django/core/management/commands/test.py

```diff
@@ -79,8 +85,11 @@ def handle(self, *test_labels, **options):
             os.environ['DJANGO_LIVE_TEST_SERVER_ADDRESS'] = options['liveserver']
             del options['liveserver']
 
+
+        tags = set(options['tags']) if options['tags'] is not None else set()
```

Comment:
Given the issue with tests and other code possibly calling `handle` directly, I think it would be best to go with something like `tags = set(options.get('tags', set()))` here.